api/proto/teleport/accesslist/v1/accesslist_service.proto

```diff
@@ -102,6 +104,9 @@ message ListAccessListMembersRequest {
 
   // page_token is the page token.
   string page_token = 2;
+
+  // access_list is the name of the access list that the member belongs to.
+  string access_list = 3;
```

Comment:
According to the comment, this is an access list name. Should we rename the field to `access_list_name` to emphasize it as the type is also very generic?

Reply:
I would push against this just because it wouldn't match the rest of the fields in this proto which have the same convention -- I just missed this one when adding it before.